src/app/reporting/Engine.h

```diff
@@ -156,12 +159,10 @@ class Engine
 
     /**
      *  mpGlobalDirtySet is used to track the dirty cluster info application modified for attributes during
```

Comment:
```suggestion
     *  mGlobalDirtySet is used to track the set of attribute/event paths marked dirty for reporting purposes.
```
or something?  I can't make sense of what "application modified for attributes" or "and further form the report" are supposed to mean....

Reply:
updated per your suggestion, thanks